src/test/java/seedu/address/logic/commands/GetCommandTest.java

```diff
@@ -0,0 +1,122 @@
+package seedu.address.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandFailure;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.address.logic.commands.CommandTestUtil.showPersonAtIndex;
+import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST_PERSON;
+import static seedu.address.testutil.TypicalParams.PARAMS_ARRAY_ALL;
+import static seedu.address.testutil.TypicalParams.PARAMS_ARRAY_FIRST;
+import static seedu.address.testutil.TypicalParams.PARAMS_ARRAY_SECOND;
+import static seedu.address.testutil.TypicalPersons.getTypicalAddressBook2;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import javafx.collections.ObservableList;
+import seedu.address.logic.Messages;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.person.Person;
+
+/**
+ * Contains integration tests (interaction with the Model) and unit tests for
+ * {@code GetCommand}.
+ */
+public class GetCommandTest {
+    private Model model;
+    private Model expectedModel;
+
+    @BeforeEach
+    public void setUp() {
+        model = new ModelManager(getTypicalAddressBook2(), new UserPrefs());
+        expectedModel = new ModelManager(model.getAddressBook(), new UserPrefs());
+    }
+
+    @Test
+    public void execute_validParamsUnfilteredList_success() {
+        GetCommand getCommand = new GetCommand(PARAMS_ARRAY_ALL);
+        ObservableList<Person> personList = expectedModel.getAddressBook().getPersonList();
+        String names = "";
+        String phoneNumbers = "";
+        String emails = "";
+        String addresses = "";
+        for (int i = 0; i < personList.size() - 1; i++) {
```

Comment:
Can convert `personList` to `Stream` and use `map` and `reduce` to build the strings. I think this way is better as you would no longer have to worry about indexing :)

Reply:
ur right ill change it to streams thnks for the suggestion !